examples/gno.land/r/gnoland/home/home.gno

```diff
@@ -182,7 +182,7 @@ func packageStaffPicks() ui.Element {
 					ui.BulletList{
 						ui.Link{URL: "r/sys/names"},
 						ui.Link{URL: "r/sys/rewards"},
-						ui.Link{URL: "r/sys/validators"},
```

Comment:
Honestly, the first time I read `vals` I thought about `values`, not `validators`

Reply:
We will use `r/sys/vars` or something like `r/sys/config/vars` for the configuration values; but I get the confusion

We'll roll with `r/sys/vals` for v1, and change it later if needed

Reply:
What about renaming it back to `validators`? I believe it's not only a problem of readability, but also a matter of pronunciation. We typically say "validator" when speaking, not "vals."

`vals` and `vars` are similar in pronunciation, and `vals` and `values` are similar in appearance. `validators` is a clear and unique term.